plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -4272,7 +4388,9 @@ public void registerCoreMembers() {
         // @Video /denizen/vids/Loops
         //
         // @Description
-        // TODO: Document Command Details
+        // Runs a series of braced commands until the tag returns false.
+        // To end a while loop, do - while stop
+        // To jump immediately to the next entry in the loop, do - while next

```

Comment:
These are written perfectly for placement as `@Usage` sections. Not so great as a description.

Reply:
While not directly relevant to the current documentation changes, I assume you'll want the current repeat documentation to be later changed too?
(Since this was a modified version from the repeat description)